prime-router/docs/proposals/0019-universal-pipeline-filter-reporting.md

```diff
@@ -0,0 +1,168 @@
+# Summary
+
+When a report satisfies a receiver’s jurisdictional filter requirements, the report still need to satisfy other filters
+before it actually gets sent to that receiver. If any of those filters return negative results, logs summarizing the
+filtered report are saved to the submission history with a description of which filters resulted in the filtered items.
+Because there are a few different situations in which a report would be filtered out at this step, this document intends
+to summarize what information is included in the filtered report logs, and why that information is included.
+
+# Details
+
+## Filter Types
```

Comment:
I was thinking that might eventually be better suited for a document focused on the filters rather than on the logging, but I suppose it doesn't hurt to include that here too.

Reply:
My point is, I don't think we need a "logging" document. I think much more valuable is to have a "UP Filters" document, with a subsection on logging as it relates to the filters. That way everything related to filters is in one place. Thoughts?

Reply:
That may be true, but feels like a reach for this ticket, so I think for now, I'll just add some filter details to this page, and if this ends up being becoming the general filter documentation down the line, then that works fine.